libcxx/test/libcxx/utilities/tuple/abi.compile.pass.cpp

```diff
@@ -0,0 +1,81 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Let's also add tests for `is_empty`.

Reply:
It turns out emptyness checks are way less interesting than I thought at first. A tuple with more than zero elements is never empty, because `__tuple_impl` is always a non-zero-size member of `tuple`.